lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1301,48 +1301,11 @@ ClangImporter::create(ASTContext &ctx,
 
   ClangInvocationFileMapping fileMapping =
     getClangInvocationFileMapping(ctx, nullptr, ignoreFileMapping);
+  applyClangInvocationMapping(ctx, fileMapping, VFS);
```

Comment:
I also prefer if we combine those two functions (or have the new function including the call to the first one).

I guess when caching is on, there is also no need to call the first function.